cmd/attack/redis.go

```diff
@@ -85,6 +86,23 @@ func NewRedisSentinelStopCommand(dep fx.Option, options *core.RedisCommand) *cob
 	return cmd
 }
 
+func NewRedisCachePenetrationCommand(dep fx.Option, options *core.RedisCommand) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cache-penetration",
+		Short: "penetrate cache",
+		Run: func(*cobra.Command, []string) {
+			options.Action = core.RedisCachePenetrationAction
+			utils.FxNewAppWithoutLog(dep, fx.Invoke(redisAttackF)).Run()
+		},
+	}
+
+	cmd.Flags().StringVarP(&options.Addr, "addr", "a", "", "")
+	cmd.Flags().StringVarP(&options.Password, "password", "p", "", "The password of server")
+	cmd.Flags().IntVarP(&options.RequestNum, "request-num", "", 0, "The number of requests")
```

Comment:
I think `0` is not a good default value.

Reply:
Because I think generating a random code is an unnecessary operation to simulate cache penetration. On the other hand, generating random code will bring additional time consumption which may make the loop runs slowly(especially when the request-num is very big). But I do accept the idea of using random codes, it's not a big deal.